### DISCUSSION TITLE
Single tap gesture conflict with live text button

### DISCUSSION THREAD

**Discussion by jfmoe** [software-mansion/react-native-gesture-handler#2879] [Q&A]
I'm using `expo-image` and enable live text on ios. When I single tap the button, it is hijacked by `GestureDetector` and can't trigger live text. What should I do to deal with the conflict?

```tsx
const tap = Gesture.Tap().onEnd(() => {
  //  ...
})

<GestureDetector gesture={tap}>
  <View style={styles.container} collapsable={false}>
    <Image enableLiveTextInteraction source={...}/>
  </View>
</GestureDetector>
```

<img width="382" alt="image" src="https://github.com/software-mansion/react-native-gesture-handler/assets/34602628/21433fde-a158-4829-b835-0257a56cb539">

**Comment by thanhlichncc** [comment]
Have you found a way to fix this problem?

**Comment by terreng** [comment]
I believe you can also use `Gesture.Tap().cancelsTouchesInView(false)` to solve it. The side effect is that it won't register taps anywhere on the image where there is text on the first try.
